### PR TITLE
feat: add status to group list and group show

### PR DESCRIPTION
**Pull request by gris**
surface new field to the CLI

Screenshot 2024-06-07 at 5 04 57 PM

Screenshot 2024-06-07 at 5 05 07 PM

the broken test doesn't look related to my code, but I'm happy to fix it later

### CODE REVIEW

File: internal/cmd/group.go [hunk 1]
```diff
@@ -210,12 +212,33 @@ func destroyGroup(client *turso.Client, name string) error {
...
 	for _, group := range groups {
-		row := []string{group.Name, formatLocations(group.Locations, group.Primary), group.Version, formatBool(group.Archived)}
+		row := []string{group.Name, formatLocations(group.Locations, group.Primary), group.Version, aggregateGroupStatus(group)}
 		data = append(data, row)
 	}
 	return data
 }
 
+func aggregateGroupStatus(group turso.Group) string {
+	status := "Healthy 🟢"
```
Comment (athoscouto) [inline internal/cmd/group.go:222]: WDYT about dropping the emojis? I don't think they add too much next to the words and they're also unpadded.

**Review by athoscouto — APPROVED** [review]
Last

File: internal/cmd/group.go [hunk 2]
```diff
@@ -210,12 +212,33 @@ func destroyGroup(client *turso.Client, name string) error {
...
+	if group.Archived {
+		return "Sleeping 💤"
+	}
+	for _, locationStatus := range group.Status.Locations {
+		if group.Primary == locationStatus.Name && locationStatus.Status == "down" {
+			status = "Busted 🔴"
+			break
+		}
+		if group.Primary == locationStatus.Name && locationStatus.Status == "stopped" {
```
[the remark this answers is not in the document]
Comment (gris) [reply internal/cmd/group.go:231]: agreed

File: internal/cmd/group.go [hunk 3]
```diff
@@ -210,12 +212,33 @@ func destroyGroup(client *turso.Client, name string) error {
...
+			status = "Idle ⚪️"
```
[the remark this answers is not in the document]
Comment (gris) [reply internal/cmd/group.go:232]: that makes sense, I always forget that one can be happy with only reads

File: internal/cmd/group.go [hunk 4]
```diff
@@ -210,12 +212,33 @@ func destroyGroup(client *turso.Client, name string) error {
...
 	var data [][]string
 	for _, group := range groups {
-		row := []string{group.Name, formatLocations(group.Locations, group.Primary), group.Version, formatBool(group.Archived)}
+		row := []string{group.Name, formatLocations(group.Locations, group.Primary), group.Version, aggregateGroupStatus(group)}
 		data = append(data, row)
 	}
 	return data
 }
 
+func aggregateGroupStatus(group turso.Group) string {
```
[the remark this answers is not in the document]
Comment (gris) [reply internal/cmd/group.go:221]: probably better, the reason I didn't put it in the api in the first place is that it is kind of a "heuristic" if we consider individual locations, but I agree that it makes more sense to be there.

**Pull request merged**